packages/bpk-component-calendar/src/Week.js

```diff
@@ -119,11 +119,28 @@ class Week extends Component {
       preventKeyboardFocus,
       selectedDate,
       showWeekendSeparator,
+      isScrollable,
     } = this.props;
 
     const firstDayOfWeekendIndex = getFirstDayOfWeekend(daysOfWeek);
     const lastDayOfWeekendIndex = getLastDayOfWeekend(daysOfWeek);
 
+    function or(total, bool) {
+      return total || bool;
+    }
+
+    if (isScrollable) {
```

Comment:
Can we rename this prop to be more descriptive of what happens inside `Week` rather than the outer concern of scrolling?